data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/command/dataexport/MultiColumnKeyValueConverter.java

```diff
@@ -0,0 +1,65 @@
+package com.scalar.db.dataloader.cli.command.dataexport;
+
+import com.scalar.db.dataloader.core.ColumnKeyValue;
+import java.util.ArrayList;
+import java.util.List;
+import picocli.CommandLine;
+
+/**
+ * Converts a string representation of multiple key-value pairs into a list of {@link
+ * ColumnKeyValue} objects.
+ *
+ * <p>The expected format for the input string is: {@code key1=value1,key2=value2,...}. Each
+ * key-value pair should be separated by a comma, and each pair must follow the "key=value" format.
+ *
+ * <p>Example usage:
+ *
+ * <pre>
+ *   MultiColumnKeyValueConverter converter = new MultiColumnKeyValueConverter();
+ *   List&lt;ColumnKeyValue&gt; result = converter.convert("name=John,age=30,city=New York");
+ * </pre>
+ */
+public class MultiColumnKeyValueConverter
+    implements CommandLine.ITypeConverter<List<ColumnKeyValue>> {
+
+  /**
+   * Converts a comma-separated string of key-value pairs into a list of {@link ColumnKeyValue}
+   * objects.
+   *
+   * @param keyValue the input string in the format {@code key1=value1,key2=value2,...}
+   * @return a list of {@link ColumnKeyValue} objects representing the parsed key-value pairs
+   * @throws IllegalArgumentException if the input is null, empty, or contains invalid formatting
+   */
+  @Override
+  public List<ColumnKeyValue> convert(String keyValue) {
```

Comment:
`keyValue` seems like a single key-value pair.

Reply:
How about moving this logic and implementation to a more generic something like `List<KeyValue> OptionParser.convert(String keyValueList)`, so that other converter classes (e.g., ScanOrderingConverter) can reuse that?

Reply:
@komamitsu san, 
The `MultiColumnKeyValueConverter` was added when a scenario of multiple clustering keys may be added.
I have updated the logic not in the way you suggested. I have moved the common logic to a util class so it can be reused without code duplication. Please take a look and let me know what you think about it.
Thank you.